docs/manage/fleet/reuse-configuration.md

```diff
@@ -357,7 +357,7 @@ Here are the version options:
 
 {{< /expand >}}
 {{< expand "Change a package version" >}}
-This example uses [`$set`](https://www.mongodb.com/docs/manual/reference/operator/update/set/#mongodb-update-up.-set) to set [version update settings for the first package in the packages array of the fragment](/data-ai/ai/deploy/#deploy-a-specific-version-of-an-ml-model) in the fragment:
+This example uses [`$set`](https://www.mongodb.com/docs/manual/reference/operator/update/set/#mongodb-update-up.-set) to modify [version update settings for the first package in the packages array of the fragment](/data-ai/ai/deploy/#deploy-a-specific-version-of-an-ml-model) from the fragment:
```

Comment:
"of the fragment from the fragment" (whether in or from) is hard to parse--assuming this isn't a typo, is there a way to restructure this sentence? Or is this actually meant to be
```suggestion
This example uses [`$set`](https://www.mongodb.com/docs/manual/reference/operator/update/set/#mongodb-update-up.-set) to modify [version update settings for the first package in the packages array of the fragment](/data-ai/ai/deploy/#deploy-a-specific-version-of-an-ml-model):
```
?